frontend/src/app/components/UpdateList/index.scss

```diff
@@ -118,10 +118,12 @@
 
     &:hover {
       border-color: $transparent-white-2;
+      background:rgba(255,255,255,0.05);
     }
 
     &:active {
-      border-color: $transparent-white-5;
+      border-color: $transparent-white-4;
+      background:rgba(255,255,255,0.08);
```

Comment:
for the background color, please add a new sass variable to this page (https://github.com/mozilla/testpilot/blob/master/frontend/src/styles/_utils.scss) then you can use the variable here!